src/ui/server/helpers/sanitise-environment/index.test.ts

```diff
@@ -22,23 +22,23 @@ describe('helper/sanitise-environment/sanitise', () => {
     expect(result).toBe('');
   });
 
-  // Returns the input string if it does not contain `\\\\n`
-  it('should return the input string if it does not contain `\\\\n`', () => {
+  // Returns the input string if it does not contain `\\\n`
+  it('should return the input string if it does not contain `\\\n`', () => {
```

Comment:
Not a blocker - I don't think we need a description of the test as the test describes what it's testing 😛 but lets get this merged